GitCommands/PathUtil.cs

```diff
@@ -19,14 +19,14 @@ public static class PathUtil
         public static readonly char PosixDirectorySeparatorChar = '/';
         public static readonly char NativeDirectorySeparatorChar = Path.DirectorySeparatorChar;
 
-        /// <summary>Replaces native path separator with posix path separator.</summary>
+        /// <summary>Replaces native path separator with posix path separator (/).</summary>
         [return: NotNullIfNotNull("path")]
         public static string? ToPosixPath(this string? path)
         {
             return path?.Replace(NativeDirectorySeparatorChar, PosixDirectorySeparatorChar);
         }
 
-        /// <summary>Replaces '\' with '/'.</summary>
+        /// <summary>Replaces '/' with native path separator.</summary>
```

Comment:
I'm not so sure about "\". IIRC we convert all paths from `C:\path\path` to `C:/path/path`.

Reply:
ToPosixPath replaces \ with / as you say.
This is ToNativePath which does the reverse.

Reply:
Thanks for checking.